src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/MinerAndroid.java

```diff
@@ -106,4 +99,23 @@ protected void moveAndDig(Block b, BlockMenu menu, BlockFace face, Block block)
         }
     }
 
+    @ParametersAreNonnullByDefault
+    private void breakBlock(BlockMenu menu, Collection<ItemStack> drops, Block block) {
+        // Push our drops to the inventory
+        for (ItemStack drop : drops) {
```

Comment:
Are we not checking if drops fit anymore? Old code had a if (menu.fits(drop, getOutputSlots())) check, I don't see why we would skip it

Reply:
I intentionally removed it because it seemed redundant. When we push items that dont fit, nothing really happens.

So checking if items don't and then doing nothing instead kinda leads to the same outcome anyway. That's why I removed it.